services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/schema/AnnotationsTranslatorImpl.java

```diff
@@ -405,9 +443,15 @@ void writeAnnotationValue(String key, AnnotationsValue value, JSONObject jsonObj
 		}
 		if (value.getValue().isEmpty()) {
 			jsonObject.put(key, "");
+		} else if (value.getValue().size() == 1 && isSingleMap.isEmpty()) {
```

Comment:
This logic will change with the addition of the new test.

Need a more complex map.  Map<Key, Enum> Where enum {SINGLE, ARRAY, NOT_DEFINED}
map.getOrElse(NOT_DEFINED);
Logic:
not_defined = Same as no schema case.
single = must be a single
array = must be an array